docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/test-api/assertions/
+checked: false
+---
+# Assertions
+
+To check if the state of the tested webpage matches the one you expect to see, use *assertions*.
+
+TestCafe provides a comprehensive [set of assertions](assertion-api.md) and all of them based on Behavior Driven Development style (BDD-style).
+
+This topic contains the following sections.
+
+* [Assertion timeout](#assertion-timeout)
+* [Assertion options](#assertion-options)
+  * [options.timeout](#optionstimeout)
+
+The following example demonstrates how to check whether two values are equal or not.
+
+```text
```

Comment:
I think it should not.
Look at other examples:
text https://github.com/DevExpress/testcafe/pull/997/files#diff-70e36160da834fc09f9ab211e6fd2d9dR29
code  https://github.com/DevExpress/testcafe/pull/997/files#diff-70e36160da834fc09f9ab211e6fd2d9dR42